### PR TITLE
[WIP] Pipeline design to facilitate multi scan processing

### PR DESCRIPTION
**Pull request by ttngu207**
Addressing #15 
Simplify the workflow side as users no longer need to provide functions to resolve suite2p or caiman dir, but instead provide the relative path in the manual table `ProcessingTask`

### CODE REVIEW

File: elements_imaging/imaging.py [hunk 1]
```diff
@@ -140,12 +138,20 @@ class MaskType(dj.Lookup):
...
+    -> Session
     -> ProcessingParamSet
-    ---
+    scan_ids: varchar(16)  # list of the scan_id associated with this ProcessingTask (delimited by '_' for example)
+    ---    
     task_mode='load': enum('load', 'trigger')  # 'load': load computed analysis results, 'trigger': trigger computation
+    processing_output_dir: varchar(255)        #  output directory relative to root data directory
     """
 
+    class MemberScan(dj.Part):
```
Comment (dimitri-yatsenko) [inline elements_imaging/imaging.py:149]: needs a comment explaining.

Comment (dimitri-yatsenko) [reply elements_imaging/imaging.py:149]: or you may rename this class to be more self explanatory e.g. `ScanInSequence` or `ScanInGroup` or `ScanToProcess`.

File: elements_imaging/imaging.py [hunk 2]
```diff
@@ -154,13 +160,19 @@ class Processing(dj.Computed):
...
 
-    class ProcessingOutputFile(dj.Part):
+    class Field(dj.Part):
         definition = """
-        -> master
-        file_path: varchar(255)  # filepath relative to root data directory
+        field_idx           : int
+        """
+
+    class MemberField(dj.Part):
```
Comment (dimitri-yatsenko) [inline elements_imaging/imaging.py:171]: Use a better name that explains the intent. E.g. `FieldToProcess`

**Comment by ttngu207** [comment]
Closing this [WIP], re-open as a new PR pointing to the new `multi-scan-processing` branch

**Pull request closed without merging**